### ISSUE TITLE
model evaluation

### ISSUE THREAD

**Issue by hankyuwon** [haofanwang/video-swin-transformer-pytorch#12]
Is it okay to write the evaluation of the model like this

model.eval()
        with torch.no_grad():
            for image_data, label in validation_dataloader:

Should I set the value of 'frozen_stages' in SwinTransformer3D?
